browse/controllers/__init__.py

```diff
@@ -4,22 +4,3 @@
 Each controller corresponds to a distinct browse feature with its own request
 handling logic.
 """
-from typing import Optional
-from flask import request
-from browse.services.database import get_institution
-
-
-def get_institution_from_request() -> Optional[str]:
-    """Get the institution name from the request context."""
-    institution_str = None
-    try:
-        institution_str = get_institution(request.remote_addr)
-
-    except IOError:
-        # TODO: log this
-        # return {
-        #     'explanation': 'Could not access the database.'
-        # }, status.HTTP_500_INTERNAL_SERVER_ERROR, {}
-        return None
-
-    return institution_str
```

Comment:
@mhl10 I moved this to the abs controller but I think we might want to move this to base since other pages will need the institution header?